pkgs/development/python-modules/jupyter-themes/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  lesscpy,
+  matplotlib,
+  notebook,
+}:
+
+buildPythonPackage rec {
+  pname = "jupyter-themes";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "dunovank";
+    repo = "jupyter-themes";
+    tag = "v${version}";
+    hash = "sha256-AWbUXMOA6k2LpZtcJOPxTZb1oL5vLDbBq4aEhXWvy9M=";
+  };
+
+  dependencies = [
+    lesscpy
+    matplotlib
+    notebook
+  ];
+
+  pythonImportsCheck = [ "jupyterthemes" ];
+
```

Comment:
add pytestCheckHook too, please

Reply:
I tried, but [the tests](https://github.com/dunovank/jupyter-themes/blob/master/tests/test_themes.py) are not written using pytest, so pytest finds nothing to test.

I also tried using nose2pytest to modernize the tests, but could not get it to work.

Actually, reading the tests more thoroughly, I don't see how they could possibly run successfully, as there are references to undefined variables (`mf`, `nf`, `tc`) in the first call to `install_themes()`...